.packit.yaml

```diff
@@ -6,3 +6,10 @@ synced_files:
   - .packit.yaml
 upstream_project_name: osbuild
 downstream_package_name: osbuild
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+      - fedora-30-x86_64
+      - fedora-rawhide-x86_64
```

Comment:
Looks perfect to me! Would be nice to put a brief description in the commit message about what the change is. And if there are unrelated changes (as I asked below), either point them out in the commit message or split them out into a separate commit (either is fine with me).

Reply:
it should by in a separate commit by now